src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1390,7 +1390,7 @@ class SelectClauseSegment(BaseSegment):
 
     type = "select_clause"
     match_grammar: Matchable = StartsWith(
-        Sequence("SELECT", Ref("WildcardExpressionSegment", optional=True)),
+        "SELECT",
```

Comment:
There is no point in ending in an optional part in `StartsWith`.

Nothing really to do with this, but noticed when working on this.